ccmlib/scylla_node.py

```diff
@@ -191,6 +191,9 @@ def _start_jmx(self, data):
         jmx_jar_dir = os.path.join(self.get_path(), BIN_DIR)
         jmx_java_bin = os.path.join(jmx_jar_dir, 'symlinks', 'scylla-jmx')
         jmx_jar = os.path.join(jmx_jar_dir, 'scylla-jmx-1.0.jar')
+        jmx_select_java = Path(jmx_jar_dir) / 'select-java'
+        if jmx_select_java.exists():
+            jmx_java_bin = jmx_select_java
```

Comment:
Nitpick: I would put the first setting of jmx_java_bin above as a "else" here. It will make it less confusing when somebody comes back to this code a year from now and doesn't understand which of the code is relevant.

Another note (slightly more serious):  I don't know why somebody called this symlink "scylla-jmx" but I suspect it was so in "top" or "killall" you'll see the string "scylla-jmx". But with your change, it will now be called "select-java", not "scylla-jmx". Perhaps if you want to do something more along the line of the original goal of this code, instead of modifying this code you should modify the code which creates this link in the first place - and link select-java to symlinks/scylla-jmx?

Reply:
I am quite sure we aren't looking at the processes by name like that, since we are saving the pid of the process and kill it with the pid.

Reply:
I've took you advice and now we symlink `select-java` into `scylla-jmx`
it's simplify the change into one place.